src/rdb.c

```diff
@@ -1169,7 +1169,6 @@ int rdbSaveInfoAuxFields(rio *rdb, int rdbflags, rdbSaveInfo *rsi) {
         if (rdbSaveAuxFieldStrInt(rdb,"repl-offset",server.master_repl_offset)
             == -1) return -1;
     }
-    if (rdbSaveAuxFieldStrInt(rdb,"aof-preamble",aof_preamble) == -1) return -1;
```

Comment:
Maybe it's a good idea to keep saving that (or other) indication in the rdb? I.e. Note that it's an rdb that's part of an AOF system? 
May come in handy some day..

Reply:
Add a new aux like ` if (rdbSaveAuxFieldStrInt(rdb,"aof-base",1) == -1) ` ?